### ISSUE TITLE
BWDC 2.8.2 not syncing users

### ISSUE THREAD

**Issue by ImplicitDeny** [bitwarden/directory-connector#92]
Since 2.8.2 is out, our LDAP sync is not working properly anymore. I'll describe symptoms hereafter. All symptoms are true with Directory Connector as the AppImage AND the CLI.
- bwdc test : displays the proper list of users (71) and groups (29).
- bwdc sync : no error, in GUI a popup says "71 users and 29 groups synced", however in the vault there is no new user (65 including those who have not yet accepted the invite). Groups are correctly synced, though.
- bwdc clear-cache : no error
- bwdc sync : same result, annonces 71 users and 29 groups, but only sync groups.

If I delete or add a new group, the next sync is properly reflecting the changes. If I delete or add a new user, the next sync detect it (so LDAP connection is fine) and announces syncing it, but in the webvault there is no new user added.

No error whatsoever, no alert, no warning in dev tools, all seems to be ok but still, it doesn't work.
Organization has currently 75 seats available, that's 4 more than needed.

Probably not related, but a strange thing happens when syncing : we have a group for tech administrators, in which I am myself. It is named "SecOps", and is properly detected in LDAP (bwdc test) with 7 users including myself. These 7 users are declared as Organization's owner, as they are allowed (and asked) to maintain collections and sync, etc. On each sync however, despite being correctly detected as members of group SecOps, all of us are removed from the group, excepted one. There is a single user, also ranked as owner, which is kept in SecOps groups, all other are removed and must be added back manually.

I'm starting to think this directory connector is haunted by some wandering soul down here...

**Comment by kspearrin** [comment]
@ImplicitDeny Are you bumping up against your user seat limit for your organization? There is a known issue with this tool that will report a successful sync, but end up doing nothing if you are trying to sync more users than your license allows for.

**Comment by ImplicitDeny** [comment]
I'm updating this in order to avoid having open issues that are fixed in between :
I reached the support, and while taking screeshots in order to show them the problem, I noticed it was indeed fixed. I asked the support if any kind of updates had been deployed, affecting BWDC. Support confirmed that an update related to directory connector was made a few days before I reach them. So far, I haven't seen any similar problem after that. I think we can consider that the problem was linked to something in an older version, and is no more causing trouble now.

Sorry @kspearrin I missed your reply's notification. The user seat limit was fine at this moment, as stated in my original message, the expected number of users after sync was 71, while the organization is capable of 75.
